packages/networks/src/defaults/genesis.ts

```diff
@@ -36,6 +36,9 @@ export const knownGenesis: KnownGenesis = {
     '0xb3db41421702df9a7fcac62b53ffeac85f7853cc4e689e0b93aeb3db18c09d82',
     '0x67dddf2673b69e5f875f6f25277495834398eafd67f492e09f3f3345e003d1b5'
   ],
+  cerenetwork: [
```

Comment:
What about just `cere`?

Reply:
Yes, it's better to use cere in this case because we have `cere` in the ss58 register: 
```
{
    "prefix": 54,
    "network": "cere",
    "displayName": "Cere Network",
    "symbols": [
      "CERE"
    ],
    "decimals": [
      10
    ],
    "standardAccount": "*25519",
    "website": "https://cere.network"
  },
  ```